com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -1,5 +1,251 @@
+- [85377d1a] (2021-04-29) Albin Corén / fix: NetworkBehaviour editor now uses owner NetworkBehaviour (#783)
+- [340324dc] (2021-04-29) Albin Corén / fix: NetworkTransform no longer uses NetworkManager.Singleton (#766)
+- [3e1aef34] (2021-04-29) Albin Corén / fix: NetworkManager doesn't destroy itself on multi instance (#765)
+- [ea307953] (2021-04-28) Noel Stephens / refactor: scene object serialization consolidation and improved handling (#756)
+- [75e09775] (2021-04-27) Albin Corén / refactor: FindObject(s)OfType now filters NetworkManagers (#763)
+- [ae70d511] (2021-04-27) Albin Corén / refactor!: NetworkBehaviour and NetworkObject NetworkManager instance can now be overriden (#762)
+- [ca676c3b] (2021-04-26) Noel Stephens / test: Companion Tests for MTT-640 and PR-749 (#754)
+- [9e127c10] (2021-04-23) Albin Corén / refactor: BufferManager now uses internal NetworkManager (#746)
+- [6998fd47] (2021-04-22) Noel Stephens / feat!: Network Prefab Overrides, Inspector View, and the default Player Prefab (#749)
+- [cd92e897] (2021-04-22) Albin Corén / fix: RpcQueueProcessor now uses the correct instance to invoke (#747)
+- [4beabfea] (2021-04-20) Albin Corén / refactor: InternalMessageSender is no longer static (#739)
+- [4a130606] (2021-04-19) Albin Corén / refactor!: NetworkSceneManager is no longer static (#738)
+- [b7357f85] (2021-04-19) Andrew Spiering / ci: Enabling MLAPI UTP Transport package (#736)
+- [ac8e4623] (2021-04-16) Albin Corén / refactor: remove version control using (#748)
+- [9dc01b00] (2021-04-16) Albin Corén / refactor!: CustomMessageManager is no longer static (#737)
+- [54a45938] (2021-04-16) Albin Corén / refactor: InternalMessageHandler is no longer static (#706)
+- [c0386eb3] (2021-04-15) Jesse Olmer / Merge branch 'master' into develop
+- [68482608] (2021-04-15) Cosmin / docs: Remove MIT license hyperlink in the main README.md (#732)
+- [644c71a1] (2021-04-14) Jesse Olmer / Merge branch 'master' into develop
+- [0f7d388b] (2021-04-14) Jesse Olmer / Merge tag '0.1.0' into develop
+- [6984f6d6] (2021-04-14) will-mearns / docs: add "expected outcome" section to bug report template (#728)
+- [8219636e] (2021-04-13) Noel Stephens / feat!: NetworkPrefabHandler to replace UsePrefabSync and Custom Spawn/Destroy Handlers (#727)
+- [52e91a45] (2021-04-08) Jeffrey Rainy / chore: Move NetworkVariable traffic onto a reliable fragmented sequen… (#717)
+- [f6cdc679] (2021-04-08) Jesse Olmer / chore: codeowners for transport, scene mgmt, and docs (#712)
+- [a52b0bf0] (2021-04-07) Jeffrey Rainy / test: Adding a first test to check the robustness of MLAPI to transpo… (#709)
+- [d9969907] (2021-04-07) kvassall-unity / chore: add CODEOWNERS (#697)
+- [e51d7e71] (2021-04-06) Albin Corén / refactor: BufferManager is no longer static (#705)
+- [9f5f39b4] (2021-04-06) Albin Corén / refactor!: SpawnManager is no longer static (#696)
+- [2571b8ce] (2021-03-31) DanZolnik / feat!: add client id to spawn handler delegate (#685)
```

Comment:
we might add this as an entry to our changelog above @DanZolnik 👀

Reply:
Sounds good, thanks!